tests/test_lib.py

```diff
@@ -0,0 +1,27 @@
+# testing whether csv is correctly written.
+
+from commons_boundaries.lib import write_csv_from_shape
+
+import csv
+import fiona
+import tempfile
+import unittest
+
+
+class TestWriteCSV(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        self.shp_path = './tests/data/italy-source/CAMERA_PLURI_2017.shp'
+        self.collection = fiona.open(self.shp_path, 'r')
+        self.fieldnames = self.collection.schema['properties'].keys()
+
+        self.csvfile = tempfile.NamedTemporaryFile(mode='w+t', suffix='.csv')
```

Comment:
I think these two lines should be in a test method rather than a `setUpClass` or `setUp` method - `write_csv_from_shape` is really the action you're taking (the middle part of the ["arrange", "act", "assert" sequence](http://wiki.c2.com/?ArrangeActAssert) in each test). This also means that you could take out the `tearDownClass` method and perhaps use a context manager for automatic closing of the written CSV file instead.

Another thing to consider here is whether you really want the creation of `self.collection` to be in `setUpClass` (run only once before all the test methods in the class) or `setUp` (run before each test method) - I don't know enough about fiona to know if self.collection is guaranteed not to be modified by what the tests are doing, but if you create it in `setUp` instead, you don't need to worry about the tests not being isolated.